rocky/bytes_client.py

```diff
@@ -25,6 +30,71 @@ def health(self) -> ServiceHealth:
 
         return ServiceHealth.parse_obj(response.json())
 
+    @staticmethod
+    def raw_from_declarations(declarations: List[Declaration]):
+        json_string = f"[{','.join([declaration.json() for declaration in declarations])}]"
+
+        return json_string.encode("utf-8")
+
+    def add_manual_proof(self, raw: bytes, manual_mime_type: str = "manual/ooi"):
+        """Per convention for a generic normalizer, we add a raw list of declarations, not a single declaration"""
+        self.login()
```

Comment:
Not for this PR, but let's improve the login logic later? 

I assume the retrieved token is valid for some time. So perhaps setting a TTL-cached property would do the job?